common/global_parameter_loader/package.xml

```diff
@@ -12,6 +12,7 @@
 
   <exec_depend>vehicle_info_util</exec_depend>
 
+  <test_depend>ament_cmake_ros</test_depend>
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>autoware_lint_common</test_depend>
 
```

Comment:
@cyn-liu It seems the test is failing because it cannot find sample_vehicle_description package.
Please do the following:
* add https://github.com/autowarefoundation/sample_vehicle_launch to [build_depends.repos](https://github.com/autowarefoundation/autoware.universe/blob/main/build_depends.repos)
* add sample_vehicle_description to package.xml as test_depend